spec/log_parser_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'log_parser'
 
-describe 'ReadFile' do
+describe 'LogParser' do
```

Comment:
```suggestion
describe LogParser do
```

O correto é usar o nome da classe aqui

Reply:
Corrigido!